caffe2/ideep/ideep_utils.h

```diff
@@ -9,12 +9,23 @@
 
 namespace caffe2 {
 
+enum FusionType {
+  FUSION_UNKNOWN = 0,
+  FUSION_CONV_RELU = 1,
+  FUSION_CONV_SUM = 2,
+  FUSION_CONV_SUM_RELU = 3,
+  FUSION_MAX = FUSION_CONV_SUM_RELU + 1,
+};
```

Comment:
Move enum from ConvFusion to here as public type.
Then, we can use it in optimize_ideep.cc